specification/securitydevops/resource-manager/Microsoft.SecurityDevOps/preview/2022-09-01-preview/securitydevops.json

```diff
@@ -0,0 +1,2692 @@
+        },
+        "reposCount": {
+          "format": "int64",
+          "description": "Gets or sets repos count.",
+          "type": "integer"
+        }
+      }
+    },
+    "GitHubOwner": {
+      "description": "GitHub repo owner Proxy Resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/GitHubOwnerProperties"
+        }
+      }
+    },
+    "GitHubOwnerListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets list of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GitHubOwner"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "Gets or sets next link to scroll over the results.",
+          "type": "string"
+        }
+      }
+    },
+    "GitHubOwnerProperties": {
+      "description": "GitHub Repo Owner properties.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "ownerUrl": {
+          "description": "Gets or sets gitHub owner url.",
+          "type": "string"
+        }
+      }
+    },
+    "GitHubRepo": {
+      "description": "GitHub repo Proxy Resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/GitHubRepoProperties"
+        }
+      }
+    },
+    "GitHubRepoListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets list of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GitHubRepo"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "Gets or sets next link to scroll over the results.",
+          "type": "string"
+        }
+      }
+    },
+    "GitHubRepoProperties": {
+      "description": "GitHub Repo properties.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "accountId": {
+          "format": "int64",
+          "description": "Gets or sets gitHub repo account id.",
+          "type": "integer"
+        },
+        "repoUrl": {
+          "description": "Gets or sets gitHub repo url.",
+          "type": "string"
+        },
+        "ownerName": {
+          "description": "Gets or sets GitHub Owner Name.",
+          "type": "string"
+        }
+      }
+    },
+    "GitHubReposProperties": {
```

Comment:
>GitHubReposProperties [](http://example.com/codeflow?start=5&length=21)

Should be GitHbRepositoryProperties or GitHubRepoProperties... (not 'Repos' plural) assuming its fine to have the github name in there.